Userland/Applications/TextEditor/MainWidget.cpp

```diff
@@ -702,6 +711,8 @@ void MainWidget::set_path(StringView path)
     if (m_extension == "c" || m_extension == "cc" || m_extension == "cxx" || m_extension == "cpp" || m_extension == "c++"
         || m_extension == "h" || m_extension == "hh" || m_extension == "hxx" || m_extension == "hpp" || m_extension == "h++") {
         m_cpp_highlight->activate();
+    } else if (m_extension == "cmake" || (m_extension == "txt" && m_name == "CMakeLists")) {
```

Comment:
Aha, ignore that previous comment (ish) and add CMakeCache.txt here :D

This is kinda... not very extensible though.

Reply:
CMakeCache.txt is its own different format, that's need its own highlighter. (Which shouldn't be complicated, but yeah)